cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java

```diff
@@ -756,4 +756,32 @@ public void testCustomTargetDirWithTests() {
                 ".json")));
     }
 
+    @Test(description = "Build an empty library module")
+    public void testBuildEmptyLibModule() throws IOException {
+        Path projectPath = this.testResources.resolve("emptyLibProject");
+        System.setProperty("user.dir", projectPath.toString());
+
+        BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, false);
+        new CommandLine(buildCommand).parse();
+        buildCommand.execute();
+
+        String buildLog = readOutput(true);
+        Assert.assertEquals(buildLog.replaceAll("\r", ""),
+                getOutput("build-empty-lib-project.txt"));
```

Comment:
Shall we rename the file to `build-empty-package.txt` to reflect the test case name?